tests/testthat/helper.R

```diff
@@ -0,0 +1,9 @@
+# When devtools::load_all is run, create symbolic link for sql directory
+# Allows testing with devtools::test
+if (Sys.getenv("DEVTOOLS_LOAD") == "true") {
+  print("setting sql folder symobolic link")
+  packageRoot <- normalizePath(system.file("..", package = "CohortMethod"))
+  # Create symbolic link so code can be used in devtools::test()
+  R.utils::createLink(link = file.path(packageRoot, "sql"), system.file("sql", package = "CohortMethod"))
```

Comment:
Dr Google says this need Administrator privileges on a Windows system.      Does it work in your hands in (1) the JnJ environment and (2) via Github Actions.

In the (recent) past (for LEGEND-T2DM) I ended up having to physically copy files as making a symbolic link required elevated privileges.

Reply:
Maybe I am being too  restrictive here ... it's just to enable `devtools::test` and not to _actually_ run a study package.